abr_control/controllers/osc.py

```diff
@@ -38,19 +43,22 @@ class OSC(controller.Controller):
     nkv : float
         derivative gain term for null controller
     """
-    def __init__(self, robot_config, kp=1, kv=None, vmax=0.5,
-                 null_control=True, use_g=True, use_C=False, use_dJ=False):
+    def __init__(self, robot_config, kp=1, kv=None, ki=0, vmax=0.5,
+                 null_control=True, use_g=True, use_C=False, use_dJ=False,
+                 int_err=None):
```

Comment:
hmmm not happy with `int_err` ... maybe we should be letting the user choose P, PD, PID, PI, ... ? oh, or always integrating error and if ``ki = 0` then it's just not added in ...

Reply:
ah, although we want to be able to load in a starting point ... hm @tbekolay what do you suggest? having this parameter feels clumsy to me, would we be better off having a function that sets the starting point for `int_err` or somesuch?

Reply:
Yeah, int_err is more so used as a saved weight, but for the integral term